### PR TITLE
Simplify dockerfile

### PR DESCRIPTION
**Pull request by bodsch**
I have tried to optimize one of the many Dockerfiles a little bit.
- using a [multi-stage build](https://docs.docker.com/develop/develop-images/multistage-build/) to download and verify that I don't have it in the final container.
- a little more order and only one RUN layer left
- VOLUME added

I don't know if it makes sense to create a separate directory for each release. 
Here you could work more effectively with build arguments.

### CODE REVIEW

File: 5.15.6-alpine/Dockerfile [hunk 1]
```diff
@@ -1,35 +1,74 @@
...
+  ACTIVEMQ_VERSION=5.15.6 \
+  ACTIVEMQ_HOME=/opt/activemq \
+  ACTIVEMQ_TCP=61616 \
+  ACTIVEMQ_AMQP=5672 \
+  ACTIVEMQ_STOMP=61613 \
+  ACTIVEMQ_MQTT=1883 \
+  ACTIVEMQ_WS=61614 \
+  ACTIVEMQ_UI=8161
+
+COPY --from=downloader /tmp/apache-activemq-$ACTIVEMQ_VERSION-bin.tar.gz  /tmp/
```
Comment (rmohr) [inline 5.15.6-alpine/Dockerfile:41]: I guess it would make sense to extract activemq in the "downloader" image.

**Comment by rmohr** [comment]
@bodsch sorry for missing your PR. I don't use the project anymore and don't have much time for it. If you want to continue this after this long time, just ping me explicitly on the PR to get my attention.

**Comment by bodsch** [comment]
> @bodsch sorry for missing your PR. I don't use the project anymore and don't have much time for it. If you want to continue this after this long time, just ping me explicitly on the PR to get my attention.

Thanks for your feedback.
I'm currently not even sure why and how I found your repository. :)

**Comment by rmohr** [comment]
Thanks you for your contribution.

As you probably noticed, I am not using this repo myself anymore for a long time and fail to find the time to maintain it for quite some time now too. Please consider using a fork. This repo is unmaintained and I am going to archive it soon.

**Pull request closed without merging**